docs/dev/contributing.md

```diff
@@ -197,19 +197,19 @@ Nautobot ChatOps currently has no intended scheduled release schedule, and will
 When a new release of any kind (e.g. from `develop` to `main`, or a release of a `stable-<major>.<minor>`) is created the following should happen.
 
 - A release PR is created:
-    - Add and/or update to the changelog in `docs/admin/release_notes/version_<major>.<minor>.md` file to reflect the changes.
-    - Update the mkdocs.yml file to include updates when adding a new release_notes version file.
-    - Change the version from `<major>.<minor>.<patch>-beta` to `<major>.<minor>.<patch>` in `pyproject.toml`.
-    - Set the PR to the proper branch, e.g. either `main` or `stable-<major>.<minor>`.
+  - Add and/or update to the changelog in `docs/admin/release_notes/version_<major>.<minor>.md` file to reflect the changes.
+  - Update the mkdocs.yml file to include updates when adding a new release_notes version file.
+  - Change the version from `<major>.<minor>.<patch>-beta` to `<major>.<minor>.<patch>` in `pyproject.toml`.
+  - Set the PR to the proper branch, e.g. either `main` or `stable-<major>.<minor>`.
```

Comment:
Do we not have readthedocs in CI for this repo? I'm hesitant about these whitespace changes, IIRC some markdown renderers need the additional indentation for nested lists.

Reply:
I don't believe that is in dev-standards, but further up in this file we have the double space instead of quadruple.

Reply:
Darn, you're right. I need to update my mdlint checks.